connection.go

```diff
@@ -139,6 +140,8 @@ type Connection struct {
 	state   uint32
 	dec     *msgpack.Decoder
 	lenbuf  [PacketLengthBytes]byte
+
+	lastStreamId uint64
```

Comment:
What does it used for?

Reply:
Just an incremental counter for stream id, the same implementation in tarantool (but in Lua).